app/assets/javascripts/app/pages/admin_pods.js

```diff
@@ -29,6 +29,16 @@ app.pages.AdminPods = app.views.Base.extend({
 
   _showMessages: function() {
     var msgs = document.createDocumentFragment();
+    if (gon.totalCount && gon.totalCount > 0) {
+      let totalPods = $("<div class='alert alert-info' role='alert' />")
+        .append(Diaspora.I18n.t("admin.pods.total", {count: gon.totalCount}));
+      msgs.appendChild(totalPods[0]);
+    }
+    if (gon.activeCount && gon.activeCount > 0) {
+      let active = $("<div class='alert alert-info' role='alert' />")
+        .append(Diaspora.I18n.t("admin.pods.active", {count: gon.activeCount}));
```

Comment:
I think there are already a lot of boxes here, I would combine these two new sentences into the same box and maybe change the wording a little. And if there are `totalCount` there should always also be `activeCount`, shouldn't it?

So how about "x pods were active recently" for the second part, and that can go into the same box after the total count? I think it's also fine to show if there were 0 active pods recently, so no need to check `activeCount > 0`.

![image](https://user-images.githubusercontent.com/458548/181870661-e6519f9c-38b8-4531-9eab-d36a6daa6d06.png)